conn.go

```diff
@@ -9,6 +9,12 @@ import (
 	"time"
 )
 
+// DefaultConnTimeout time in seconds to wait for connection to beanstalk server.
+const DefaultConnTimeout = 10
```

Comment:
IMO it would be a little better for this to be a time.Duration instead of an untyped number.

Also, I think it would be better to name it DefaultDialTimeout, since "dial timeout" is more specific than "conn timeout" and better conveys that it only applies to establishing the conn, and not for the rest of the conn's lifetime.

Something like:

```suggestion
// DefaultDialTimeout is the time to wait for a connection to the beanstalk server.
const DefaultDialTimeout = 10 * time.Second
```